src/cli/cli.rs

```diff
@@ -72,8 +74,22 @@ impl Debug for dyn SubCommand {
 /// Trace packets on the Linux kernel
 ///
 /// packet-tracer is a tool for capturing networking-related events from the system using ebpf and analyzing them.
-#[derive(Args, Default, Debug)]
-pub(crate) struct MainConfig {}
+#[derive(Args, Debug, Default)]
+pub(crate) struct MainConfig {
+    #[arg(
+        long,
+        default_value = "warn",
+        help = "Log level: error, warn, info, debug. Default: warn."
+    )]
+    pub(crate) log_level: String,
+    #[arg(
+        short,
+        long,
+        default_value = "false",
+        help = "Suppress stdout messages."
+    )]
+    pub(crate) quiet: bool,
```

Comment:
Does it make sense a global --quite flag? what does `retis --quiet analyze`  mean? I'm thinking maybe this is more a `collect` option linked to `--out` as already exposed in another comment.